jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -95,8 +99,10 @@ private JaegerTracer(
     this.metrics = metrics;
     this.zipkinSharedRpcSpan = zipkinSharedRpcSpan;
     this.scopeManager = scopeManager;
+    this.baggageRestrictionManager = baggageRestrictionManager;
```

Comment:
so, like, a bug? 👍

Reply:
No actually just wasn't needed outside of constructor. Originally needed this when I had constructor `CustomTracer(JaegerTracer tracer)`, but don't need this anymore. Will remove.

Reply:
remove